File: medium/pro-carrot-cake/CarrotCake.py
CAKE = 7

carrots = int(input())
boxes = int(input())

left_over = carrots % boxes

if left_over < CAKE:
    need_more = CAKE - left_over
    print(f'I need to buy {need_more} more')
else:
    print('Cake Time')

File: medium/pro-carrot-cake/CarrotCake.rb
CAKE = 7
carrots = gets.chomp.to_i
boxes = gets.chomp.to_i

left_over = carrots % boxes

if left_over < CAKE
    need_more = CAKE - left_over
    puts 'I need to buy %d more' % need_more
else
    puts 'Cake Time'
end
